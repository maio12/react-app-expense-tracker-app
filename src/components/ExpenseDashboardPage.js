import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboardPage = () => (
    <div>
        This is from the dashboard component
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);

export default ExpenseDashboardPage;
