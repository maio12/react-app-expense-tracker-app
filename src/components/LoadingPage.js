import React from 'react';


const LoadingPage = () => (
    <div>
        <img className="loader__image" src="/public/img/loader.gif" alt="loader gif" />
    </div>
);

export default LoadingPage;