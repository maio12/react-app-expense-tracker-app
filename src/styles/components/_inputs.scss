.text-input {
    border: 1px solid rgb(240, 240, 240);
    height: 50px;
    padding: $s-size;
    font-size: $font-size-large;
    font-weight: 300;
}

.select {
    @extend .text-input;
}

.textarea {
    @extend .text-input;
    height: 10rem;
}