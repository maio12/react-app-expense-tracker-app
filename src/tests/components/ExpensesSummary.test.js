import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render expensesSumary with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={245}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should render expensesSumary with more expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={2} expensesTotal={400}/>);
    expect(wrapper).toMatchSnapshot();
});