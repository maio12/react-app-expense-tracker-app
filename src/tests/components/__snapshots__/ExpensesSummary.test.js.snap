// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensesSumary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     for a total of 
    $2.45
  </h1>
</div>
`;

exports[`should render expensesSumary with more expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     for a total of 
    $4.00
  </h1>
</div>
`;
